tasks/Module 4/task_1.py

```diff
@@ -0,0 +1,27 @@
+"""Mad libs."""
+import os
+import re
+
+TEMPLATE = ['ADJECTIVE', 'NOUN', 'ADVERB', 'VERB']
```

Comment:
I would rather use a different name for it eg. `PLACEHOLDERS`. The name `TEMPLATE` sounds like something that contains placeholders and you will change them to get final result, eg. content of the source file is a template in my understanding.